go.mod

```diff
@@ -1,7 +1,7 @@
 module github.com/jzelinskie/faq
 
 require (
-	github.com/Azure/draft v0.16.0
+	github.com/Azure/draft v0.16.0 // indirect
```

Comment:
It doesn't look like you got rid of this. I don't think we should get rid of it anyway because linguist is able to autodetect formats like TOML, which your naive matching doesn't work on.

Reply:
That basically means throwing away streaming for auto detection and adding a good bit of complexity, but I can try.

Reply:
Is that true? We don't need to give linguist the whole file.